src/main/java/seedu/address/logic/commands/EditCommand.java

```diff
@@ -82,13 +79,11 @@ public class EditCommand extends Command {
             + PREFIX_ALLERGY + "nuts, shellfish "
             + PREFIX_HEALTHRISK + "HIGH "
             + PREFIX_EXISTINGCONDITION + "Diabetes "
-            + PREFIX_APPOINTMENT + "2022-12-31T14:00"
             + PREFIX_NOTE + "Patient needs extra care";
 
     public static final String MESSAGE_EDIT_PATIENT_SUCCESS = "Edited Patient: %1$s";
     public static final String MESSAGE_NOT_EDITED = "At least one field to edit must be provided.";
     public static final String MESSAGE_DUPLICATE_PATIENT = "This patient already exists in the address book.";
-    public static final String WRONG_NRIC = "NRIC provided has to be a current valid NRIC in the system.";
```

Comment:
Why are we removing the Wrong NRIC message string?

Reply:
This is available in Messages.java because we can use it for multiple commands - EditCommand, DeleteCommand, ApptCommand